app/views/unattended/provisioning_templates/provision/autoyast_default.erb

```diff
@@ -9,9 +9,11 @@ description: |
 -%>
 <%
   # safemode renderer does not support unary negation
+  aio_enabled = host_param_true?('enable-puppetlabs-repo') || host_param_true?('enable-puppet5') || %w[official-puppet7 puppetlabs-puppet6 puppetlabs-puppet5].any? { |repo| host_param_true?("enable-#{repo}-repo") }
```

Comment:
Here we have still a reference to puppet5. Should we get rid of all the puppet5 stuff?

Reply:
Definitely sounds reasonable to throw out the EoL versions